src/patterns/dosdp-dev/abnormalShapeOfPhysicalEntityInLocation.yaml

```
@@ -0,0 +1,53 @@
+---
+pattern_name: abnormalShapeOfPhysicalEntityInLocation
+pattern_iri: http://purl.obolibrary.org/obo/upheno/patterns-dev/abnormalShapeOfPhysicalEntityInLocation.yaml
+description: "abnormal shape of an anatomical entity or a measurement of a process, such as an ECG wave, in a location"
+
+contributors:
+- https://orcid.org/0000-0002-3528-5267
+- https://orcid.org/0000-0001-5208-3432
+- https://orcid.org/0000-0003-4148-4606
+
+classes:
+  shape: PATO:0000052
+  abnormal: PATO:0000460
+  entity: BFO:0000002
+  independent continuant: BFO:0000004
+
+relations:
+  inheres_in: RO:0000052
+  has_modifier: RO:0002573
+  has_part: BFO:0000051
+
+annotationProperties:
+  exact_synonym: oio:hasExactSynonym
+
+vars:
+  entity: "'entity'"
+  location: "'independent continuant'"
+
+name:
+  text: "abnormal shape of %s in %s"
+  vars:
+  - entity
+  - location
+
+annotations:
+- annotationProperty: exact_synonym
+  text: "abnormally shaped %s in %s"
+  vars:
+  - entity
+  - location
```

```suggestion
  - annotationProperty: exact_synonym
    text: "abnormally shaped %s in %s"
    vars:
      - entity
      - location
```
* yaml syntax error    wrong indentation: expected 2 but found 0  (indentation)